src/chart/options/channel.h

```diff
@@ -63,7 +63,7 @@ static_assert(std::ranges::all_of(Refl::enum_names<LegendId>,
 	            Refl::get_enum<ChannelId>(name));
     }));
 
-[[nodiscard]] constexpr Geom::Orientation operator+(
+[[nodiscard]] constexpr Geom::Orientation operator++(
```

Comment:
`++` still not intuitive, I think it make sence to convert between Axis and Channel ID with an operator, since they are the same set of values, only that Axis is a subset.
However Orientation and AxisID are different, I would use a conversion function instead, IMHO it would make the code more readable.